tools/docker-compose.yml

```diff
@@ -68,11 +68,7 @@ services:
       - CELERY_ACKS_LATE=true
       - REDIS_USE_SENTINEL=false
       - CELERYD_MAX_TASKS_PER_CHILD=4096
-      - SMALL_COMPARISON_CHUNK_SIZE=100000000
-      - LARGE_COMPARISON_CHUNK_SIZE=1000000000
-      - SMALL_JOB_SIZE=100000000
-      - LARGE_JOB_SIZE=100000000000
-      - MATCH_THRESHOLD=0.98
```

Comment:
Why did you remove the default threshold?

Reply:
Same reason - it isn't a used environment variable so it is just misleading people who might change it expecting something about our software to change...